R/utils-hub_tasks_config.R

```diff
@@ -42,3 +58,48 @@ is_target_ordinal <- function(task_groups_w_target) {
   target_type <- task_groups_w_target[[1]]$target_metadata[[1]]$target_type
   return(target_type == "ordinal")
 }
+
+
+#' Get the output type id values for a given output type. The output type may
+#' appear in multiple task groups, and the output type id values in those groups
+#' may differ as long as there is one group that has all of the output type id
+#' values.
+#' @noRd
+get_output_type_ids_for_type <- function(task_groups, output_type) {
+  output_type_ids_by_group <- purrr::map(
+    task_groups,
+    function(task_group) {
+      task_group$output_type[[output_type]]$output_type_id
+    }
+  )
+
+  output_type_ids <- output_type_ids_by_group[[which.max(lengths(output_type_ids_by_group))]]
```

Comment:
I was a little unclear of what this was doing, but seeing the tests made it clear, so I thought it warranted a comment. 

```suggestion
  # Small groups should contain subsets of the largest group, so this is our reference. 
  output_type_ids <- output_type_ids_by_group[[which.max(lengths(output_type_ids_by_group))]]
```